petpals/forum/post/templates/post.html

```diff
@@ -37,8 +37,17 @@ <h5 class="mt-1">{{ post.title }}</h5>
               </div>
               <div class="text-center">
                 <button class="btn btn-xs btn-outline-info">
-                  <a href="{{ url_for('forum_router.post_router.reply_router.reply_new', post_id=post.post_id) }}">Reply</a>
+                  <a
+                    href="{{ url_for('forum_router.post_router.reply_router.reply_new', post_id=post.post_id) }}">Reply</a>
                 </button>
+                {% if post.user_id == current_user.id %}
+                <button class="btn btn-xs btn-outline-info">
+                  <a href="{{ url_for('forum_router.post_router.edit_post', post_id=post.post_id)}}">Edit Post</a>
+                </button>
+                <button class="btn btn-xs btn-outline-info">
+                  <a href="{{ url_for('forum_router.post_router.delete_post', post_id=post.post_id)}}">Delete Post</a>
+                </button>
```

Comment:
Delete post should be on `edit_post.html`, please refer to `edit_reply.html`.

Reply:
@EthanPintoA Can you do this one?  I know what your wanting, but I keep getting all of the errors b/c I don't understand how to work with the modal.  I know you can probs do this in 2 seconds, and I will spend HOURS on it lmao

Reply:
Sure